samples/gen-go/client/doer.go

```diff
@@ -91,3 +99,196 @@ func (d retryDoer) Do(c *http.Client, r *http.Request) (*http.Response, error) {
+		Num50  int `json:"50"`
+		Num75  int `json:"75"`
+		Num90  int `json:"90"`
+		Num95  int `json:"95"`
+		Num99  int `json:"99"`
+		Num100 int `json:"100"`
+		Nine95 int `json:"99.5"`
+	} `json:"latencyTotal"`
+	CircuitBreakerRequestVolumeThreshold             int    `json:"propertyValue_circuitBreakerRequestVolumeThreshold"`
+	CircuitBreakerSleepWindowInMilliseconds          int    `json:"propertyValue_circuitBreakerSleepWindowInMilliseconds"`
+	CircuitBreakerErrorThresholdPercentage           int    `json:"propertyValue_circuitBreakerErrorThresholdPercentage"`
+	CircuitBreakerForceOpen                          bool   `json:"propertyValue_circuitBreakerForceOpen"`
+	CircuitBreakerForceClosed                        bool   `json:"propertyValue_circuitBreakerForceClosed"`
+	CircuitBreakerEnabled                            bool   `json:"propertyValue_circuitBreakerEnabled"`
+	ExecutionIsolationStrategy                       string `json:"propertyValue_executionIsolationStrategy"`
+	ExecutionIsolationThreadTimeoutInMilliseconds    int    `json:"propertyValue_executionIsolationThreadTimeoutInMilliseconds"`
+	ExecutionIsolationThreadInterruptOnTimeout       bool   `json:"propertyValue_executionIsolationThreadInterruptOnTimeout"`
+	ExecutionIsolationThreadPoolKeyOverride          string `json:"propertyValue_executionIsolationThreadPoolKeyOverride"`
+	ExecutionIsolationSemaphoreMaxConcurrentRequests int    `json:"propertyValue_executionIsolationSemaphoreMaxConcurrentRequests"`
+	FallbackIsolationSemaphoreMaxConcurrentRequests  int    `json:"propertyValue_fallbackIsolationSemaphoreMaxConcurrentRequests"`
+	MetricsRollingStatisticalWindowInMilliseconds    int    `json:"propertyValue_metricsRollingStatisticalWindowInMilliseconds"`
+	RequestCacheEnabled                              bool   `json:"propertyValue_requestCacheEnabled"`
+	RequestLogEnabled                                bool   `json:"propertyValue_requestLogEnabled"`
+}
+
+func logEvent(l *logger.Logger, e HystrixSSEEvent) {
+	l.InfoD(e.Name, map[string]interface{}{
+		"requestCount":                    e.RequestCount,
+		"errorCount":                      e.ErrorCount,
+		"errorPercentage":                 e.ErrorPercentage,
+		"isCircuitBreakerOpen":            e.IsCircuitBreakerOpen,
+		"rollingCountFailure":             e.RollingCountFailure,
+		"rollingCountFallbackFailure":     e.RollingCountFallbackFailure,
+		"rollingCountFallbackSuccess":     e.RollingCountFallbackSuccess,
+		"rollingCountShortCircuited":      e.RollingCountShortCircuited,
+		"rollingCountSuccess":             e.RollingCountSuccess,
+		"rollingCountThreadPoolRejected":  e.RollingCountThreadPoolRejected,
+		"rollingCountTimeout":             e.RollingCountTimeout,
+		"currentConcurrentExecutionCount": e.CurrentConcurrentExecutionCount,
+		"latencyTotalMean":                e.LatencyTotalMean,
+	})
+
+}
+
+func (d *circuitBreakerDoer) init() {
+	// Turn on the SSE port + listener once
+	circuitSSEOnce.Do(func() {
```

Comment:
this is a bit wonky. is there a chance we could make a PR against hysterix-go for an API that exposes these?

otherwise, i guess we don't have another option